nav/about-data/style.css

```diff
@@ -0,0 +1,11 @@
+.nowrap {
+    white-space: nowrap;
+}
+
+.em-zh {
```

Comment:
No, so I fixed this using `em:lang(zh)`.

I also removed some `em`s in the Chinese translation, because Chinese rarely adds emphasis marks to a long sentence/paragraph (like the descriptions of groups in https://www.w3.org/International/i18n-drafts/nav/about#groups ).